src/utils/validators.ts

```diff
@@ -0,0 +1,20 @@
+export const isRequired = (inputName: string, value: string) => {
+    if (!value || value.length === 0) {
```

Comment:
may be worth adding a trim to make sure it's not just whitespace characters?
```suggestion
    if (!value || value.trim().length === 0) {
```